doc/src/manual/strings.md

```diff
@@ -617,20 +617,21 @@ julia> "1 + 2 = 3" == "1 + 2 = $(1 + 2)"
 true
 ```
 
-You can search for the index of a particular character using the [`findfirst`](@ref) function:
+You can search for the index of a particular character using the
+[`findfirst`](@ref) and [`findlast`](@ref) functions:
 
 ```jldoctest
-julia> findfirst(isequal('x'), "xylophone")
-1
+julia> findfirst(isequal('o'), "xylophone")
+4
 
-julia> findfirst(isequal('p'), "xylophone")
-5
+julia> findlast(isequal('o'), "xylophone")
+7
 
 julia> findfirst(isequal('z'), "xylophone")
 ```
 
-You can start the search for a character at a given offset by using [`findnext`](@ref)
-with a third argument:
+You can start the search for a character at a given offset by using
+[`findnext`](@ref) and [`findprev`](@ref) with a three arguments:
```

Comment:
"a three"

Maybe just remove the mention of the number of arguments, that's clear just by looking at the examples.